string class_name
RGB color 
uint32 x0
uint32 y0
uint32 h
uint32 w
float32 distance
float32 certainty
